### PR TITLE
[map-sort-by-value] sort map by value

### PR DESCRIPTION
**Pull request by sk1418**

### CODE REVIEW

File: core-java-modules/core-java-collections-maps-7/src/test/java/com/baeldung/map/LinkedHashMapSortByValueUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        EXPECTED_PLAYERS.put("player a", new Player("Eric", 9));
+        EXPECTED_PLAYERS.put("player c", new Player("Amanda", 20));
+    }
+
+    @Test
+    void whenUsingCollectionSort_thenGetExpectedResult() {
+        List<Map.Entry<String, Integer>> entryList = new ArrayList<>(MY_MAP.entrySet());
+        Collections.sort(entryList, new Comparator<Map.Entry<String, Integer>>() {
+            @Override
+            public int compare(Map.Entry<String, Integer> o1, Map.Entry<String, Integer> o2) {
```
Comment (ashleyfrieze) [inline core-java-modules/core-java-collections-maps-7/src/test/java/com/baeldung/map/LinkedHashMapSortByValueUnitTest.java:101]: Can't we do this as a lambda function?

**Review by ashleyfrieze — CHANGES_REQUESTED** [review]

File: core-java-modules/core-java-collections-maps-7/src/test/java/com/baeldung/map/LinkedHashMapSortByValueUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        return name;
+    }
+
+    public int getScore() {
+        return score;
+    }
+}
+
+public class LinkedHashMapSortByValueUnitTest {
+    private static LinkedHashMap<String, Integer> MY_MAP = new LinkedHashMap();
```
Comment (ashleyfrieze) [inline core-java-modules/core-java-collections-maps-7/src/test/java/com/baeldung/map/LinkedHashMapSortByValueUnitTest.java:58]: The type on the right hand side is missing its `<>`

File: core-java-modules/core-java-collections-maps-7/src/test/java/com/baeldung/map/LinkedHashMapSortByValueUnitTest.java (hunk 1 above)
Comment (sk1418) [reply core-java-modules/core-java-collections-maps-7/src/test/java/com/baeldung/map/LinkedHashMapSortByValueUnitTest.java:101]: The article has two parts, one for Java < ver 1.8. The other is for 1.8+. 
This code is used in the section with version earlier than Java 1.8.

File: core-java-modules/core-java-collections-maps-7/src/test/java/com/baeldung/map/LinkedHashMapSortByValueUnitTest.java (hunk 2 above)
Comment (sk1418) [reply core-java-modules/core-java-collections-maps-7/src/test/java/com/baeldung/map/LinkedHashMapSortByValueUnitTest.java]: oh, yes, diamonds are missing. Fixed in the new commit.
Thank you.

**Pull request merged**